python/tests/test_speech_content_extract.py

```diff
@@ -0,0 +1,215 @@
+    assert "Dr. Vogel" in df["name_raw"].values
+    assert "Bundesminister der Justiz" in df["position_raw"].values
+    assert "Huonker" in df["name_raw"].values
+    assert "Staatsminister beim Bundeskanzler" in df["position_raw"].values
+    assert "Gallus" in df["name_raw"].values
+    assert "Parl. Staatssekretär beim Bundesminister für Ernährung, Landwirtschaft und Forsten" in df["position_raw"].values # because the name is written differently, it causes issues in the position_raw
+    assert "Frau Merkel" in df["name_raw"].values
+    assert "Bundeskanzlerin" in df["position_raw"].values
+    assert "Frau Schulz" in df["name_raw"].values
+    assert "Schriftführerin" in df["position_raw"].values
+    assert "Fischer" in df["name_raw"].values
+    assert "Senator" in df["position_raw"].values
+    # assert "Zinn" in df["name_raw"].values
+    # assert "(SPD)" in df["constituency"].values
+    # assert "Berichterstatter" in df["position_raw"].values
+
+def test_faction_speaker_extraction_pre_term10(tmp_path):
+    # Arrange
+    # Create the parent folder
+    electoral_term_dir = tmp_path / "electoral_term_pp08"
+    electoral_term_dir.mkdir(exist_ok=True)
+
+    # Create the subfolder
+    session_dir = electoral_term_dir / "08008"
+    session_dir.mkdir(exist_ok=True)
+
+    # Create the session content file inside the subfolder
+    session_file = session_dir / "session_content.txt"
+    session_file.write_text(
+        "Vizepräsidentin Frau Renger: Bitte, fahren Sie fort.\n"
+        "Sund (SPD): Die Bundesregierung, sagten Sie, habe die Arbeitslosigkeit verharmlost. "
+        "Sozialdemokraten, Herr Kollege Katzer, wissen, was Arbeitslosigkeit bedeutet. Warum wollen wir denn "
+        "ein Recht auf Arbeit? Das erreichen wir aber nicht mit plakativen Forderungen, sondern durch eine "
+        "mühsame und praktische Politik, in der es darum geht, Zug um Zug die Vollbeschäftigung herzustellen, "
+        "und dies unter Bedingungen, die nicht einfach sind. Das wissen Sie so gut wie wir.\n"
+        "(Zurufe von der CDU/CSU)\n"
+    )
+
+    term_number = int(session_dir.stem[:2])  # '08' -> 8
+    print(term_number) # not the issue
+    open_brackets, close_brackets, prefix = get_bracket_and_prefix_from_term_number(session_dir.parent)
+    print(open_brackets, close_brackets, prefix)
+
+    faction_speaker_pattern = get_faction_speaker_pattern(
+        term_number,
+        session_dir,
+        open_brackets,
+        close_brackets,
+        prefix
+    )
+
+    patterns = [faction_speaker_pattern]
+    save_path = tmp_path / "output"
+    save_path.mkdir()
+
+    # Act
+    process_session(session_dir, patterns, save_path)
+
+    # Assert
+    df = pd.read_pickle(save_path / "08008.pkl")
+    assert "session" in df.columns
+    assert "name_raw" in df.columns
+    assert "position_raw" in df.columns
+    assert "constituency" in df.columns
+    assert "speech_content" in df.columns # Problem mit diesen Test
```

Comment:
what do you mean with "Problem"? The other test further down works? Where you assert the actual content of the speech_content column?